src/firefox/remote.js

```diff
@@ -0,0 +1,102 @@
+/* @flow */
+import {createLogger} from '../util/logger';
+import {WebExtError} from '../errors';
+import defaultFirefoxConnector from 'node-firefox-connect';
```

Comment:
whoops! I extracted this from the bigger patch so that it would be easier to review